### PR TITLE
Lab1 Proposal/fix

### PR DESCRIPTION
**Pull request by unaihuete93**
# Module: 1
## Lab/Demo: Agile Planning and Portfolio Management with Azure Boards

Changes proposed in this pull request:

- Make WIT experience easier, students get lost with dashboard widget option (not frequently used that way from experience)
- Clarify what an ellipsis is
- Move task 5 to optional exercise:  Not enough time to finish all lab in 60 minutes and this task is more related to reporting. Dashboards may have not been explained yet in module 1.
-
-

### CODE REVIEW

**Review by mkennten — COMMENTED** [review]
I like this - it's helping with some confusion and is shortening the lab by removing the creation of a dashboard from the "mandatory" exercise and leaving it as optional. Creating a dashboard is covered in another lab and not strictly related to Azure Boards.

**Comment by unaihuete93** [comment]
Any update needed? just fixed another small typo :)

File: Instructions/Labs/AZ400_M01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md [hunk 1]
```diff
@@ -327,43 +325,7 @@ To maximize a team's ability to consistently deliver high quality software, Kanb
...
-1.  On the **Editor** tab of **Queries > My Queries** pane, in the second row, in the **Field** column, select **Area Path** and, in the corresponding **Value** dropdown list, select **Agile Planning and Portfolio Management with Azure Boards\\PUL-Web**.
-1.  Click **Save query**.
-1.  In the **New query** panel, in the **Enter name** textbox, type **Web tasks**, in the **Folder** dropdown list, select to **Shared Queries**, and click **OK**.
-1.  Select the **Charts** tab and click **+ New chart**.
-1.  On the **Configure Chart** panel, in the **Name** textbox, type **Web tasks - By assignment**, in the **Group by** dropdown list, select **Assigned To**, and click **OK** to save the changes.
-
-    > **Note**: You can now add this chart to a dashboard.
-
-#### Task 6: Customize team process
+#### Task 5: Customize team process
```
Comment (ClementinoDeMendonca-msft) [inline Instructions/Labs/AZ400_M01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md:328]: Normally I make this one the optional lab. What is the rationale for making the Dashboard one instead? Or maybe both should also be optional, for the time allotted.

Comment (mkennten) [reply Instructions/Labs/AZ400_M01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md:328]: I'm going with this change for now, we can still change the customize part optional later on.

**Pull request merged**